test/integration/custom-extensions/components/world.jsx

```diff
@@ -0,0 +1,3 @@
+export const World = () => (
```

Comment:
The filepath  for this test contains `custom-extensions` text. Since this is just JSX, it'd simply write a testcase in the `basic`test cases.